File: runtime/Java/src/main/java/org/antlr/v5/runtime/tree/RuleNode.java
/*
 * Copyright (c) 2012-present The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD 3-clause license that
 * can be found in the LICENSE.txt file in the project root.
 */

package org.antlr.v5.runtime.tree;

import org.antlr.v5.runtime.RuleContext;

public interface RuleNode extends ParseTree {
	RuleContext getRuleContext();
}

File: runtime/Core/src/main/kotlin/org/antlr/v5/runtime/core/tree/RuleNode.kt
/*
 * Copyright (c) 2012-present The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD 3-clause license that
 * can be found in the LICENSE.txt file in the project root.
 */

package org.antlr.v5.runtime.core.tree

import org.antlr.v5.runtime.core.context.RuleContext

public interface RuleNode : ParseTree {
  public val ruleContext: RuleContext
}
